server/src/main/java/io/unitycatalog/server/persist/utils/FileOperations.java

```diff
@@ -196,33 +202,81 @@ private static URI adjustFileUri(URI fileUri) {
+   *
+   * <p>If the input is a valid URI with a recognized scheme (e.g., "file", "s3", "abfs", etc.), the
+   * method returns a standardized version of the URI. If the input is not a valid URI, it treats
+   * the input as a local file path and converts it to a "file://" URI.
+   *
+   * @param inputPath the input path or URI to be standardized.
+   * @return the standardized URI string.
+   * @throws BaseException if the input path has an unsupported URI scheme.
+   *     <p>Examples of input and output:
+   *     <pre>
+   * // Local File System Example:
+   * "file:/tmp/myfile"         -> "file:///tmp/myfile"
+   *
+   * // AWS S3 Example:
+   * "s3://my-bucket/my-file"   -> "s3://my-bucket/my-file"
+   *
+   * // Azure Blob Storage Example:
+   * "abfs://my-container@my-storage.dfs.core.windows.net/my-file"
+   *                          -> "abfs://my-container@my-storage.dfs.core.windows.net/my-file"
+   *
+   * // Google Cloud Storage Example:
+   * "gs://my-bucket/my-file"   -> "gs://my-bucket/my-file"
+   *
+   * // Invalid Path Example (treated as a file path):
+   * "/local/path/to/file"      -> "file:///local/path/to/file"
+   *
+   * // Unsupported Scheme Example:
+   * "ftp://example.com/file"   -> Throws BaseException with message: "Unsupported URI scheme: ftp"
+   * </pre>
+   */
+  public static String toStandardizedURIString(String inputPath) {
+    try {
+      // Check if the path is already a URI with a valid scheme
+      URI uri = new URI(inputPath);
+      // If it's a file URI, standardize it
+      if (uri.getScheme() != null) {
+        if (uri.getScheme().equals(Constants.URI_SCHEME_FILE)) {
+          return adjustLocalFileURI(uri).toString();
+        } else if (Constants.SUPPORTED_CLOUD_SCHEMES.contains(uri.getScheme())) {
+          return uri.toString();
+        } else {
+          throw new BaseException(
+              ErrorCode.INVALID_ARGUMENT, "Unsupported URI scheme: " + uri.getScheme());
+        }
+      }
+    } catch (URISyntaxException e) {
+      // Not a valid URI, treat it as a file path
     }
-    URI normalized = uri.normalize();
-    if (!normalized.getPath().startsWith(uri.getPath())) {
-      throw new BaseException(ErrorCode.INVALID_ARGUMENT, "Normalization failed: " + uri.getPath());
+    String localUri = Paths.get(inputPath).toUri().toString();
+    if (!inputPath.endsWith("/") && localUri.endsWith("/")) {
+      // A special case where the local inputPath is a directory already exist, generated localUri
+      // will have an extra trailing slash. Remove it to make it consistent.
+      localUri = localUri.substring(0, localUri.length() - 1);
     }
+    return localUri;
   }
 
-  public static void assertValidLocation(String location) {
-    validateURI(URI.create(location));
+  private String getManagedTablesStorageRoot() {
+    // Use local tmp directory as default storage root
+    return serverProperties.getProperty("storage-root.tables", "file:///tmp");
```

Comment:
Do we need to align the default value that we've declared here : https://github.com/unitycatalog/unitycatalog/pull/1134/files#diff-903b566690a83bff41179c53cc9eca441e8bd8d371e4ddfb028ce9fe0ee30af8R38 ?